tests/__init__.py

```diff
@@ -0,0 +1,17 @@
+import json
+from django.contrib.auth.models import User
+from rest_framework.test import APITestCase
+
+from wallet.models import Category, Expense
+
+
+class TomatoTest(APITestCase):
+    def setUp(self):
+        self.user_info = {'username': 'tomato', 'password': 'top_secret'}
+        self.user = User.objects.create_user(**self.user_info)
+
+        self.sb_info = {'username': 'somebody', 'password': 'top_secret'}
+        self.sb = User.objects.create_user(**self.sb_info)
```

Comment:
maybe rename it to `anonymous_user`

Reply:
the somebody just means another user, doesn't means anonymous. On the other hand, anonymous does not have to store to the database. This dict is used to create a user.